microreactors/hpmr_h2/steady/tests

```diff
@@ -0,0 +1,74 @@
+[Tests]
+  # Generate the meshes
+  [neutronics_coarse_mesh]
+    type = CheckFiles
+    input = one_twelfth_simba_core_CM.i
+    cli_args = '--mesh-only'
+    executable_pattern = 'griffin*|blue_crab*|dire_wolf*'
+    check_files = one_twelfth_simba_core_CM_in.e
+  []
+  [neutronics_fine_mesh]
+    type = CheckFiles
+    input = one_twelfth_simba_core.i
+    cli_args = '--mesh-only'
+    check_files = one_twelfth_simba_core_in.e
+    executable_pattern = 'griffin*|blue_crab*|dire_wolf*'
+    prereq = neutronics_coarse_mesh
+  []
+
+  # Syntax checks
+  [coupled_multiphysics_syntax]
+    type = RunApp
+    input = 'neutronics_eigenvalue.i'
+    check_input = True
+    executable_pattern = 'blue_crab*|dire_wolf*|fenix*'
```

Comment:
is there sockeye in fenix??

Reply:
no idea what fenix is 😆 (I was just copy pasting but I can remove it)

Reply:
it s @pbalest 's app.
It s Griffin + Pgh. Should be removed

Reply:
fixed